relay-event-normalization/src/event.rs

```diff
@@ -977,6 +978,41 @@ pub fn normalize_performance_score(
     version
 }
 
+// Extracts lcp related tags from the trace context
+pub fn normalize_trace_context_tags(event: &mut Event) {
+    let tags = &mut event.tags.value_mut().get_or_insert_with(Tags::default).0;
+    if let Some(contexts) = event.contexts.value() {
+        if let Some(trace_context) = contexts.get::<TraceContext>() {
+            if let Some(data) = trace_context.data.value() {
+                if let Some(lcp_element) = data.lcp_element.value() {
+                    if tags.get("lcp.element").is_none() {
+                        let tag_name = "lcp.element".to_string();
+                        tags.insert(tag_name, Annotated::new(lcp_element.to_string()));
+                    }
+                }
+                if let Some(lcp_size) = data.lcp_size.value() {
+                    if tags.get("lcp.size").is_none() {
+                        let tag_name = "lcp.size".to_string();
+                        tags.insert(tag_name, Annotated::new(lcp_size.try_into().unwrap()));
+                    }
+                }
+                if let Some(lcp_id) = data.lcp_id.value() {
+                    let tag_name = "lcp.id".to_string();
+                    if tags.get("lcp.id").is_none() {
+                        tags.insert(tag_name, Annotated::new(lcp_id.to_string()));
```

Comment:
This does not make a difference but it clarifies the intention (same below):

```suggestion
                        tags.insert(tag_name, Annotated::new(lcp_id.clone()));
```

Reply:
updated `lcp_id`, `lcp_element`, and `lcp_url` to us clone instead! Kept `lcp_size` as `to_string` since it's not originally a String type